changelog.d/12597.removal

```diff
@@ -0,0 +1,2 @@
+Remove the unspecified `m.login.jwt` login type and the unstable `uk.half-shot.msc2778.login.application_service` from 
```

Comment:
Do we want to shout a bit louder about the fact we're *actually* removing `JWT` support?

Reply:
> Do we want to shout a bit louder about the fact we're _actually_ removing `JWT` support?

We're not, we're only removing the unspecced `m.login.jwt` type. There's still `org.matrix.login.jwt` type as documented at https://matrix-org.github.io/synapse/develop/jwt.html

If anyone is using this they should be able to `s/m.login.jwt/org.matrix.login.jwt/g` and it should work fine.

What do you think?